src/day07/README.md

```diff
@@ -0,0 +1,14 @@
+# Kth from the end of a linked list
+
+- Given a linked list
+- Methods created in the LinkedList class can be used.
+
+## Challenge
+
+- Write a function for the linked list class that will accept a number (k) as a parameter and return the node from the index of k (the index starts at the end of the list).
+
```

Comment:
Does the index start counting at `0` like an array or `1` like normal numbers? This is an important piece of info to offer your users.